lib/action_view/component/test_helpers.rb

```diff
@@ -4,7 +4,9 @@ module ActionView
   module Component
     module TestHelpers
       def render_inline(component, **args, &block)
-        Nokogiri::HTML(ApplicationController.new.view_context.render(component, args, &block))
+        controller = ApplicationController.new
```

Comment:
I would love for this to be extracted to a separate method so that one could stub methods on the controller instance.  e.g.:
```ruby
      def controller
        @controller ||= ApplicationController.new.tap { |c| c.request = ActionDispatch::TestRequest.create }
      end
```

Reply:
I've updated this as per your suggestion - is that what you meant?